metals/src/main/scala/scala/meta/internal/metals/MetalsLanguageServer.scala

```diff
@@ -1249,6 +1250,24 @@ class MetalsLanguageServer(
             val msg = s"Invalid arguments: $args. Expecting: $argExample"
             Future.failed(new IllegalArgumentException(msg)).asJavaObject
         }
+      case ServerCommands.NewScalaWorksheet() =>
+        val args = params.getArguments.asScala
+        def f(dir: URI, name: String) = Future {
```

Comment:
We should use a better name than just `f`

Reply:
`createWorksheet` now